_data/projects/wattpad-scraper.yml

```diff
@@ -0,0 +1,12 @@
+name: Your Project Name Here
```

Comment:
@shhossain This should be `wattpad-scraper`

Reply:
name changed